docs/backend/index.md

```diff
@@ -12,7 +12,7 @@ html_meta:
 
 ```{toctree}
 :maxdepth: 2
-
+controlpanels
```

Comment:
Please rename the referenced file to `control-panel.md` for SEO, as well as this suggestion.

```suggestion
control-panels
```